src/es/components/organisms/registerForm/RegisterForm.js

```diff
@@ -253,15 +327,6 @@ export default class RegisterForm extends Shadow() {
           color: var(--m-gray-600);
           font-size: 14px;
       }
-      :host .form-radio-group {
-        display: flex;
-        gap: 2rem;
-      }
-      :host .form-radio-group span {
-        display: flex;
-        align-items: center;
-        gap: 0.5rem;
-      }
```

Comment:
Moved these lines to web components toolbox
-> https://github.com/mits-gossau/web-components-toolbox/pull/140/files#diff-4ff7151ce55b1628a0454de3c3c339eeae89a70e0297fa38f2a9fe946ec37419R190-R198

Reply:
Lets talk about this today please, I would have some question related this. Thanks

Reply:
Sure, I have already pushed this change on Form.js but I forgot to create a PR. Thanks and thanks for PR